File: android/src/main/java/com/flutterplayer/fplayer/FConst.java
package com.flutterplayer.fplayer;

public class FConst {
    static final String FF_CHANNEL = "befovy.com/fijk";
    static final String FF_PLAYER_CHANNEL = "befovy.com/fijkplayer";

    private FConst() {}

    static String getMethodChannel() {
        return FF_CHANNEL;
    }

    static String getEventChannel() {
        return FF_CHANNEL + "/event";
    }

    static String getPlayerMethodChannel(int id) {
        return FF_PLAYER_CHANNEL + "/" + id;
    }

    static String getPlayerEventChannel(int id) {
        return FF_PLAYER_CHANNEL + "/event/" + id;
    }
}

File: ios/Classes/FConst.m
#import "FConst.h"

#define FF_CHANNEL              @"befovy.com/fijk"
#define FF_PLAYER_CHANNEL(p)    @"befovy.com/fijkplayer" # p


@implementation FConst {
}

- (instancetype)init {
     self = [super init];
     return self;
}

+ (NSString *) getMethodChannel {
    return [FF_CHANNEL stringByAppendingString:@""];
}

+ (NSString *) getEventChannel {
    return [FF_CHANNEL stringByAppendingString:@"/event"];
}

+ (NSString *) getPlayerMethodChannel:(NSString *)pid {
    return [FF_PLAYER_CHANNEL(/) stringByAppendingString:pid];
}

+ (NSString *) getPlayerEventChannel:(NSString *)pid {
    return [FF_PLAYER_CHANNEL(/event/) stringByAppendingString:pid];
}

@end